gadget/ondisk.go

```diff
@@ -35,6 +35,15 @@ type OnDiskStructure struct {
 	// Node identifies the device node of the block device.
 	Node string
 
+	// DiskIndex is the index of the structure on the disk - this should be
+	// used instead of "Index" for an OnDiskStructure, "Index" comes from the
```

Comment:
The comment needs an update, we don't have Index anymore

Reply:
good catch, updated